src/Sylius/Bundle/UiBundle/DependencyInjection/SyliusUiExtension.php

```diff
@@ -59,6 +60,14 @@ private function loadEvents(array $eventsConfig, ContainerBuilder $container): v
             }
 
             foreach ($blocksPriorityQueue->toArray() as $details) {
+                if (\is_array($details['context'])) {
+                    foreach ($details['context'] as $key => &$value) {
+                        if (\is_string($value) && 0 === strpos($value, '@=')) {
```

Comment:
I wonder if there's any other way to mark expressions - something like `!php/const` in Symfony's YAML config.

Reply:
I basically took this from: https://github.com/symfony/dependency-injection/blob/bb3e33b2c67b021b72e1200c4b1319af935b4f28/Loader/YamlFileLoader.php#L869
It's the way it's implemented in Symfony atm, not the nicest I agree.
So it's also likely that it won't work with other types of configuration, `xml` for example.

Reply:
The conversion to `Expression` objects here is also not strictly necessary, as the `ExpressionLanguage` should also be able to evaluate straight from `string`. However doing it here should be slightly more efficient, as it will hopefully parse the expression during compile time and cache it.